compact_str/src/repr/mod.rs

```diff
@@ -232,7 +238,7 @@ impl Repr {
     }
 
     #[inline]
-    pub fn from_box_str(s: Box<str>) -> Self {
+    pub fn from_box_str(s: Box<str>, should_inline: bool) -> Self {
```

Comment:
IMO this can be forwarded to `Repr::from_string`

Reply:
Good call! I made this change